cmd/kubeops/main.go

```diff
@@ -182,3 +187,15 @@ func main() {
 	log.Info("All requests have been served. Exiting")
 	os.Exit(0)
 }
+
+// Returns the user agent to be used during calls to the chart repositories
+// Examples:
+// kubeops/devel
+// kubeops/2.3.4 (kubeapps v2.3.4-beta4)
+func userAgent() string {
```

Comment:
Ah, it is the same approach as we are currently doing in other components. Great!

Reply:
Yep, though I've now updated so it's a function with args, rather than relying on the global params (so it's simple to test).